tardis/utilities/executors/sshexecutor.py

```diff
@@ -28,48 +48,67 @@ async def _establish_connection(self):
         return await asyncssh.connect(**self._parameters)
 
     @property
-    async def ssh_connection(self):
+    @a.contextmanager
```

Comment:
Is there any difference between `async_context_manager` from `aiotools`, which is an implicit dependency (could be explicit instead) of `asyncssh` and `a.contextmanager`? In case you do not need the package `asyncstdlib` for probing.

Reply:
We can keep `asyncstdlin`, since you need the dependency anyhow for the probing code.